### PR TITLE
[aws-load-balancer-controller] Add automountServiceAccountToken to load balancer controller

### PR DESCRIPTION
**Pull request by joshuastern**
### Description of changes

Add ability to specify automountServiceAccountToken for the aws-load-balancer-controller service account

### Checklist
- [X] Added/modified documentation as required (such as the `README.md` for modified charts)
- [X] Incremented the chart `version` in `Chart.yaml` for the modified chart(s)
- [X] Make sure the title of the PR is a good description that can go into the release notes
- [X] Manually tested. Describe what testing was done in the testing section below

### Testing

Manual chart installation

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by joshuastern** [comment]
@M00nF1sh @kishorj bump

**Comment by kishorj** [comment]
@joshuastern thanks for your contribution.

Could you provide more details on your use case to support configuring automountServiceAccountToken?

**Comment by joshuastern** [comment]
> @joshuastern thanks for your contribution.
> 
> Could you provide more details on your use case to support configuring automountServiceAccountToken?

As part of following best practices, we want to be able to explicitly configure this field rather than get it for free. When the automountServiceAccountToken is not set, a service account token is still mounted.

**Comment by joshuastern** [comment]
@kishorj can i get an approval if everything looks good?

File: stable/aws-load-balancer-controller/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-load-balancer-controller
 description: AWS Load Balancer Controller Helm chart for Kubernetes
-version: 1.1.7
+version: 1.2.0
```
Comment (kishorj) [inline stable/aws-load-balancer-controller/Chart.yaml:4]: lets have the version as 1.1.8 instead

**Review by kishorj — COMMENTED** [review]
Thanks for the changes. I have suggested a minor change. Looks good otherwise.

**Review by kishorj — APPROVED** [review]

File: stable/aws-load-balancer-controller/Chart.yaml (hunk 1 above)
Comment (joshuastern) [reply stable/aws-load-balancer-controller/Chart.yaml]: updated

**Review by kishorj — APPROVED** [review]

**Pull request merged**